test/cluster/object_store/test_backup.py

```diff
@@ -60,7 +60,7 @@ async def test_simple_backup(manager: ManagerClient, s3_server):
            'experimental_features': ['keyspace-storage-options'],
            'task_ttl_in_seconds': 300
            }
-    cmd = ['--logger-log-level', 'snapshots=trace:task_manager=trace']
+    cmd = ['--logger-log-level', 'snapshots=trace:task_manager=trace:api=info']
```

Comment:
Isn't log level even for API set to "info" by default?

Reply:
nope, it is `warn` or `error`